### ISSUE TITLE
yukon_migrate_url_alias_node migration doesn't

### ISSUE THREAD

**Issue by leeomara** [ytgov/yukon-ca#636]
When going through the steps defined in the README, the  `migration_complete_w3.sh`script migration doesn't migrate (this runs the `yukon_migrate_url_alias_node` migration).

When run (either on it's own, as part of a group, or through `migration_complete_w3.sh`), it doesn't create or update any records.

Output typically looks like:

    [notice] Processed 0 items (0 created, 0 updated, 0 failed, 0 ignored) - done with 'yukon_migrate_url_alias_node'

When I compare with `migrate:status`, the same migration shows many records to process.

```
drush migrate:status --group=legacy_url_alias --format=table
 ------------------------------------- ------------------------------ -------- ------- ---------- ------------- --------------- --------------------- 
  Group                                 Migration ID                   Status   Total   Imported   Unprocessed   Message Count   Last Imported        
 ------------------------------------- ------------------------------ -------- ------- ---------- ------------- --------------- --------------------- 
  legacy_url_alias (legacy_url_alias)   yukon_migrate_url_alias_node   Idle     53654   52997      0             0               2024-10-17 10:30:59  
 ------------------------------------- ------------------------------ -------- ------- ---------- ------------- --------------- --------------------- 
```

**Comment by leeomara** [comment]
As a workaround, I'll try to rollback the `yukon_migrate_url_alias_node` migration and re-run it again.

**Comment by leeomara** [comment]
As a workaround, running this on UAT:

    drush migrate:rollback yukon_migrate_url_alias_node

and then

    drush migrate:import --group=legacy_url_alias --continue-on-failure

created 52997 items.

**Comment by leeomara** [comment]
w3creatives suggested that d10-queries.sql should truncate the `migrate_map_yukon_migrate_url_alias_node` table.

**Comment by leeomara** [comment]
Tested this on dev and it worked well.

**Comment by w3creatives** [comment]
Can this issue be closed?

**Comment by leeomara** [comment]
This was fixed as of #640.
